packages/ui-library/src/components/notifications/stories.js

```diff
@@ -0,0 +1,106 @@
+import Notifications, { Notification, notificationClassNameMap, notificationsClassNameMap, notificationsIconMap } from ".";
+import { keys } from "lodash";
+
+export default {
+	title: "2. Components/Notifications",
+	component: Notification,
+	argTypes: {
+		children: { control: { disable: true } },
+		position: {
+			options: [ "bottom-center", "bottom-left", "top-center", "inherit" ],
+			type: "select",
+			description: "The position of the notification. Notifications prop.",
+		},
+		id: { control: "text" },
+		variant: { options: keys( notificationClassNameMap.variant ), type: "select" },
+		size: { options: keys( notificationClassNameMap.size ), type: "select" },
+		title: { control: "text", type: "string" },
+		description: { control: "text", type: "string" },
+		onDismiss: { control: { disable: false } },
+		autoDismiss: { type: "number", description: "Miliseconds for Notification to dissapear." },
+		dismissScreenReaderLabel: { control: "text", type: "string" },
+	},
+	args: {
+		title: "Notification title",
+		description: "Notifiation description",
+		id: "notification-single",
+		dismissScreenReaderLabel: "Dismiss",
+
+	},
+	parameters: {
+		docs: {
+			description: {
+				component: "The Notifications component shows notifications on a specified position on the screen. Switch `position` in the control panel to view.",
+			},
+		},
+	},
+};
+
+const Template = ( args ) =>
+	// eslint-disable-next-line no-negated-condition
+	<div className={ args.position !== "inherit" ? "yst-fixed yst-left-0" : "" }>
```

Comment:
As mentioned above: this is not needed if you move `yst-fixed` to the individual position classes.

P.S. You could've flipped the ternary order, and then the eslint disabled is not needed 😉

Reply:
Cool, thanks :)